### PR TITLE
Set default EVM version to Istanbul.

### LINKED ISSUE #7706: Set istanbul as default EVM version

### PR DESCRIPTION
**Pull request by chriseth**
Fixes https://github.com/ethereum/solidity/issues/7706

### CODE REVIEW

File: test/libsolidity/gasTests/abiv2.sol [hunk 1]
```diff
@@ -18,7 +18,7 @@ contract C {
 //   executionCost: 1167
 //   totalCost: 1123767
 // external:
-//   a(): 530
```
Comment (chriseth) [inline test/libsolidity/gasTests/abiv2.sol:21]: Ah, it's just the sload increase!

**Review by leonardoalt — APPROVED** [review]

**Review by ekpyron — APPROVED** [review]
Maybe we should open an issue for running the gas tests for different evm versions.

File: test/libsolidity/gasTests/abiv2.sol (hunk 1 above)
Comment (axic) [reply test/libsolidity/gasTests/abiv2.sol:21]: Yeah, but we should have decreases due to the net gas metering in other places.

Comment (chriseth) [reply test/libsolidity/gasTests/abiv2.sol:21]: We don't do net gas metering because it can only be done on a transaction level.

**Pull request merged**